client_plugin/utils/config/config.go

```diff
@@ -155,6 +156,7 @@ func InitConfig(defaultConfigPath string, defaultLogPath string, defaultDriver s
 	}
 	configFile := flag.String("config", defaultConfigPath, "Configuration file path")
 	driverName := flag.String("driver", "", "Volume driver")
+	sharedDriverSlave := flag.String("sharedDriverSlave", "", "Driver which creates internal volumes")
```

Comment:
the name is not semantically accurate. Just call it storageVolimeDriver or internalVolumeDriver something like that